src/types/globals/Window.js

```diff
@@ -87,6 +89,7 @@ declare class Window {
     userAgent: string,
     platform: string,
   };
+  postMessage: (message: any, targetOrigin: string) => void;
```

Comment:
I guess this is needed for the test to be able to postMessage?

Reply:
Yeah, for some reason this API isn't typed.